CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v0.8.0
+
+* zql: add the `put` proc that adds or updates fields using a computed
```

Comment:
```suggestion
* zql: add the `put` processor that adds or updates fields using a computed
```
Like in days of Jut, I'm partial to saying "processor" in the user-facing context when possible.